### ISSUE TITLE
Question regarding partitioning / Feature Request

### ISSUE THREAD

**Issue by throwaway217** [deadhead420/arch-linux-anywhere#361]
I'm pretty sure this is intended behaviour but can you add a way to also use chosen filesystem type for the root partition during install ?

**Comment by pnedkov** [comment]
You **can** choose the file system for all partitions except swap of course. Here are the supported file systems:
![image](https://cloud.githubusercontent.com/assets/6212427/26740356/0cc6144a-478a-11e7-8071-f5582722a34d.png)

**Comment by throwaway217** [comment]
Yes I know but for example if I choose automatic partitioning and btrfs, /dev/sda1 or the root partition not the /home one is formatted using ext4

**Comment by pnedkov** [comment]
#363

With this fix you will be able to format the boot partition with Btrfs but only if you have selected manual partitioning. The auto partitioning option always formats the boot partition with ext4 regardless of the file system you have selected for the root partition. I have tested it with GRUB on a non-UEFI system and it works. Please test it.

**Comment by throwaway217** [comment]
I apologize.
I was looking at a machine on which I installed archlinux without archlinux anywhere that had a seperate root partition ( I just noticed archlinux-anywhere's automatic partitioning doesn't create one ). When I installed it I must have mistakenly formatted that as ext4. When installing archlinux-anywhere last night on my laptop I notived mkfs.ext4 /dev/bla and I concluded that archlinux-anywhere formats the /root partition ext4 regardless of chosen option without actually verifying my statement.

I don't really have a problem with the /boot partition being ext4. I am deeply sorry for wasting your time.
